README.md

```diff
@@ -89,7 +89,7 @@ fn main() -> Result<(), GooseError> {
         )
         .execute()?
         .print();
-    
+
```

Comment:
Please remove all non-related changed from this PR. White space cleanups are welcome, but belong in their own PR so we don't accidentally merge unintended changes.